setup.py

```diff
@@ -322,18 +322,7 @@ def run_setup(with_extensions):
         python_requires=">=3.6, <4",
         packages=["numcodecs", "numcodecs.tests"],
         classifiers=[
-            'Development Status :: 4 - Beta',
-            'Intended Audience :: Developers',
-            'Intended Audience :: Information Technology',
-            'Intended Audience :: Science/Research',
-            'License :: OSI Approved :: MIT License',
-            'Programming Language :: Python',
-            'Topic :: Software Development :: Libraries :: Python Modules',
-            'Operating System :: Unix',
-            'Programming Language :: Python :: 3',
-            'Programming Language :: Python :: 3.6',
-            'Programming Language :: Python :: 3.7',
-            'Programming Language :: Python :: 3.8',
+            "Programming Language :: Python :: 3.9",
```

Comment:
did you meant to delete all of those ?

Reply:
Whoops. Definitely didn't mean to delete these.

Reply:
Ahhhh my editor changed `'` to `"`, hope thats okay.